### PR TITLE
Add ThemePreviewCard component for theme dashboard

### PR DESCRIPTION
**Pull request by NeenaDev**
Feature Issue: https://github.com/vanilla/vanilla/issues/9824
mockup link: https://app.zeplin.io/project/5d8a5a6cacd3a86143741c38/screen/5da09782ddcc77305268390a

Components for theme dashboard page-
Add `<ThemePreviewCard />`  component.
Add `<CurrentThemeInfo />` component
Add `<AddTheme />` component.

### CODE REVIEW

File: library/src/scripts/theming/ThemePreviewCard.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        return (
+            <React.Fragment>
+                <div style={containerStyle} className={classes.container}>
+                    <div className={classes.wrapper}>
+                        <div style={titlebarStyle} className={classes.titlebar}>
+                            <ul className={classes.titleBarNav}>
+                                <li style={titleBarLinks} className={classes.titleBarLinks}></li>
+                                <li style={titleBarLinks} className={classes.titleBarLinks}></li>
+                                <li style={titleBarLinks} className={classes.titleBarLinks}></li>
+                            </ul>
```
Comment (alex-mtl) [inline library/src/scripts/theming/ThemePreviewCard.tsx]: This 3 hardcoded lines looks suspicious to me. If I understand this code right that is header for 3 columns?
In any case I would prefer to see `const columns = 3;` or `props.columns` or anything else and kind loop to build n-number of repeated elements with the same style and same class.

**Comment by slafleche** [comment]
@NeenaDev is this still a WIP or you want me to review it?

File: library/src/scripts/theming/ThemePreviewCard.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                                                <p className={classes.text1}></p>
+                                                <p className={classes.text2}></p>
+                                                <p className={classes.text3}></p>
+                                            </div>
+                                        </div>
+                                    </li>
+                                ))}
+                            </ul>
+                        </div>
+                        <div className={this.props.isActiveTheme ? classes.noOverlay : classes.overlay}>
```
Comment (slafleche) [inline library/src/scripts/theming/ThemePreviewCard.tsx]: Move to root i.e. classes.container. It's weird to have it a child of the body.

File: library/src/scripts/theming/themeCardStyles.ts [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+        margin: "44px 0",
+    });
+    const overlay = style("overlay", {
+        position: "absolute",
+        top: 0,
+        backgroundColor: "rgba(103, 105, 109, 0.8)",
+        opacity: 0,
+    });
+    const wrapper = style("wrapper", {
+        opacity: 1,
```
Comment (slafleche) [inline library/src/scripts/theming/themeCardStyles.ts]: Put a little transition on there. You can use `...defaultTransition("opacity")`

File: library/src/scripts/theming/ThemePreviewCard.tsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+        const containerStyle = {
+            backgroundColor: `${globalBg}`,
+        };
+
+        const headerStyle = {
+            backgroundColor: globalPrimary,
+        };
+
+        return (
+            <React.Fragment>
```
Comment (slafleche) [inline library/src/scripts/theming/ThemePreviewCard.tsx]: Unused fragment.

File: library/src/scripts/theming/currentThemeInfo.story.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+const story = storiesOf("Theme", module);
+
+story.add("Current Theme", () => {
+    return (
+        <>
+            <StoryContent>
+                <StoryHeading depth={1}>Current Theme</StoryHeading>
+
+                <CurrentThemeInfo
```
Comment (slafleche) [inline library/src/scripts/theming/currentThemeInfo.story.tsx]: Where's the image on the right with the current theme?

File: library/src/scripts/theming/AddTheme.tsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+export default class AddTheme extends React.Component<IProps> {
+    constructor(props) {
+        super(props);
+    }
+
+    public render() {
+        const classes = currentThemeClasses();
+        return (
+            <React.Fragment>
```
Comment (slafleche) [inline library/src/scripts/theming/AddTheme.tsx]: Get rid of unused fragment. We we navigating somewhere when we click on the "+"? If not, then this is a button, not a link. To disable clicking in storybook, you can wrap it with a div that has "pointer-events: none"

**Pull request merged**